fe/fe-core/src/main/java/com/starrocks/alter/SchemaChangeJobV2.java

```diff
@@ -554,9 +554,8 @@ protected void runRunningJob() throws AlterCancelException {
             LOG.info("schema change tasks not finished. job: {}", jobId);
             List<AgentTask> tasks = schemaChangeBatchTask.getUnfinishedTasks(2000);
             for (AgentTask task : tasks) {
-                if (task.getFailedTimes() >= 3) {
-                    throw new AlterCancelException(
-                            "schema change task failed after try three times: " + task.getErrorMsg());
+                if (task.isFailed() || task.getFailedTimes() >= 3) {
```

Comment:
isFailed() means only failed one time?

Reply:
> isFailed() means only failed one time?

`isFailed()` means whether the FE handle finish alter task successful, and alter task still retry three times in BE.